vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -84,4 +84,9 @@ NS_SWIFT_NAME(BraveAds)
 
 @end
 
+NS_SWIFT_NAME(BraveLogger)
+@interface BATBraveRewardsLogger : NSObject
```

Comment:
Should live in its own file and not in `BATBraveAds.h`

Reply:
Fixed